packages/scheduler/packaging

```diff
@@ -1,5 +1,5 @@
 # abort script on any command that exits with a non zero value
 set -e
 
-cp -R scheduler/db ${BOSH_INSTALL_TARGET}
-cp scheduler/target/scheduler-*.war ${BOSH_INSTALL_TARGET}
+cp -R autoscaler/scheduler/db ${BOSH_INSTALL_TARGET}
```

Comment:
is the move of the scheduler component back compatible e.g exisitng  bosh scheduler vms continue to work ?